subprojects/build-init/src/main/java/org/gradle/buildinit/plugins/internal/JavaProjectInitDescriptor.java

```diff
@@ -105,9 +102,6 @@ private void addTestFramework(BuildInitTestFramework testFramework, BuildScriptB
                     ).testRuntimeOnlyDependency(
                     "Use JUnit Jupiter Engine for testing.",
                     "org.junit.jupiter:junit-jupiter-engine:" + libraryVersionProvider.getVersion("junit-jupiter")
```

Comment:
If junit-jupiter is new enough, could we switch to the simpler declaration?

Reply:
Yes, `testImplementation("org.junit.jupiter:junit-jupiter")` is enough but it did not exist prior to `5.4.0`.